cmd/finch/pass_init_native.go

```diff
@@ -0,0 +1,25 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build linux
+
+package main
+
+import (
+	"github.com/gopasspw/gopass/pkg/pwgen"
+	"github.com/runfinch/finch/pkg/command"
+)
+
+func (pia *passInitAction) initGpgKey() command.Command {
+	passphrase := pwgen.GeneratePassword(passphraseLength, true)
```

Comment:
Since we're not even asking users to enter a passphrase, can't we do this behind the scenes? Should we instead have a mechanism that initializes the cred store whenever a user modifies their `creds_helpers` to include `pass`?

Reply:
deletion is problematic,
i would prefer not taking responsibility of key management, if its done with a command customer has responsibility to manage key lifecycle.

Reply:
We can still have the commands, but if the commands require 0 user input, I don't see the harm in doing it for them

Reply:
i still feel if we create it the lifecycle is on us to manage for example key rotation etc. if it is automatically done finch has to won the management of the key.

Reply:
What's the purpose of key rotation for a key where we don't even care about the password?

Reply:
if the key is compromised, we probably want to rotate it, and having set time to rotate it probably makes it harder to breach within certain time period.